src/api/metadataApi.js

```diff
@@ -15,8 +15,9 @@ const metadataApi = {
         if (res && id in res.data) {
           return res.data[id];
         }
+        return undefined;
       })
-      .catch(error => {
+      .catch(_error => {
```

Comment:
can't you do:

```suggestion
      .catch(() => {
```

?

Reply:
I'd rather keep the variable, so that we feel more inclined to properly handle these errors in the future